budget/config/scipoolprod.yaml

```diff
@@ -0,0 +1,14 @@
+# budget rules per user by synapse team id
+teams:
+  '3412678': # ampad-workinggroup
+    amount: '2000'
+    period: ANNUALLY
+    unit: USD
+    community_manager_emails:
+      - 'sc-support@sagebase.org'
```

Comment:
Maybe add `it@sagebase.org` to this list as well, at least in the near term.

Reply:
As I understand it, sc-support replaces that email address. @zaro0508 please confirm.